Makefile

```diff
@@ -26,11 +26,34 @@ test: ## Run the application tests
 
 .PHONY: lint
 lint: ## Run the linters
-	@flake8 .
+	@flake8 securedrop_client tests
+
+.PHONY: safety
+safety: ## Runs `safety check` to check python dependencies for vulnerabilities
+	pip install --upgrade safety && \
+		for req_file in `find . -type f -name '*requirements.txt'`; do \
+			echo "Checking file $$req_file" \
+			&& safety check --full-report -r $$req_file \
+			&& echo -e '\n' \
+			|| exit 1; \
+		done
+
+# Bandit is a static code analysis tool to detect security vulnerabilities in Python applications
+# https://wiki.openstack.org/wiki/Security/Projects/Bandit
+.PHONY: bandit
+bandit: ## Run bandit with medium level excluding test-related folders
+	pip install --upgrade pip && \
+        pip install --upgrade bandit!=1.6.0 && \
+	bandit -ll --recursive . --exclude tests,.venv
 
 .PHONY: check
 check: clean lint mypy test ## Run the full CI test suite
 
+.PHONY: update-pip-requirements
+update-pip-requirements: ## Updates all Python requirements files via pip-compile.
+	pip-compile --generate-hashes --output-file dev-requirements.txt dev-requirements.in
```

Comment:
`dev-requirements.in` should only have things that are dev specific. These two lines should be:

```bash
pip-compile --generate-hashes --output-file dev-requirements.txt dev-requirements.in requiremints.in
pip-compile --generate-hashes --output-file requirements.txt requiremints.in
```

Unless our PyPI mirror doesn't work with that somehow?

Reply:
ah so the situation here is:
 
* `dev-requirements.in` does contain only dev requirements BUT you make a fair point re: combining dev/prod requirements (i.e. so that devs only need to install one requirements file), updated in bcfd40b
* both the `requirements*txt` (require... mints? sounds delicious! 💚) files generated by pip-compile use upstream source tarballs/wheels from PSF's PyPI. `build-requirements.txt` will be the file that contains the prod requirements that point to FPF's PyPI mirror (uses a separate file so that bumping requirements here in this repo doesn't get blocked on action over in the securedrop-debian-packaging repo to update the FPF PyPI mirror).